src/errors/database-error.ts

```diff
@@ -1,14 +1,17 @@
-import type { CommonErrorProperties, ErrorOptions } from './base-error';
+import type { CommonErrorProperties, SequelizeErrorOptions } from './base-error';
```

Comment:
Shouldn't this be ErrorOptions? Since that has `cause`, SequelizeErrorOptions has `stack`. Or am I reading it wrong?

Reply:
`DatabaseError` doesn't accept a `cause` option because its first parameter is the cause:

![image](https://user-images.githubusercontent.com/1280915/161697587-4b92e037-5c33-4a55-9524-319490329382.png)